.travis.yml

```diff
@@ -21,6 +21,8 @@ matrix:
       env: INTEGRATION_TEST=react-percy-storybook PERCY_PROJECT=percy/react-percy-storybook PERCY_TOKEN="$STORYBOOK_PERCY_TOKEN"
     - node_js: "8"
       env: INTEGRATION_TEST=react-percy PERCY_PROJECT=percy/react-percy PERCY_TOKEN="$REACT_PERCY_TOKEN"
+    # - node_js: "8"
+    #   env: INTEGRATION_TEST=create-react-app PERCY_PROJECT=percy/react-percy-create-react-app PERCY_TOKEN="$CREATE_REACT_APP_PERCY_TOKEN"
```

Comment:
We can uncomment this once we add a new percy token for `create-react-app`. As a test, I tried sending the `create-react-app` snapshots to the `react-percy` project, so you can see it working in action [here](https://percy.io/percy/react-percy/builds/319857).